contracts/test/subcall.ts

```diff
@@ -4,15 +4,68 @@ import * as oasis from '@oasisprotocol/client';
 import * as cborg from 'cborg';
 import { SubcallTests } from '../typechain-types/contracts/tests/SubcallTests';
 import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/dist/src/signer-with-address';
-import { parseEther } from 'ethers/lib/utils';
+import { arrayify, parseEther } from 'ethers/lib/utils';
 import { BigNumber, BigNumberish, ContractReceipt } from 'ethers';
+import { getRandomValues, randomInt } from 'crypto';
+
+import { execSync } from 'child_process';
+
+async function sapphireDockerName() {
```

Comment:
This is not sapphire-related actually. Let's use "Docker" literal for these functions instead, because one can always simply run oasis-net-runner locally without using Docker.

```suggestion
async function getDockerName() {
```